test/jasmine/tests/gl3d_plot_interact_test.js

```diff
@@ -568,6 +568,99 @@ describe('Test gl3d plots', function() {
         .then(done);
     });
 
+    it('@gl should set the camera projection type to perspective if the camera.projection.type is not set', function(done) {
+        Plotly.plot(gd, {
+            data: [{
+                type: 'scatter3d',
+                x: [1, 2, 3],
+                y: [2, 3, 1],
+                z: [3, 1, 2]
+            }],
+            layout: {
+                scene: {
+                    camera: {
+                    }
+                }
+            }
+        })
+        .then(delay(20))
+        .then(function() {
+            expect(gd._fullLayout.scene.camera.projection.type === 'perspective').toBe(true);
+        })
+        .then(function() {
+            expect(gd._fullLayout.scene._scene.glplot.camera._ortho).toBe(false);
```

Comment:
No need for a 2nd `.then(function() { ...`, bring this `expect()` up just below the other one.